bundles/AdminBundle/Resources/public/css/admin.css

```diff
@@ -1184,7 +1184,7 @@ span.warning {
 .pimcore_editable_wysiwyg {
     position:relative;
     padding: 10px;
-    min-height: 80px;
+    height: max(100%, 100px);
```

Comment:
```suggestion
    min-height: 80px;
    height: 100%;
```

Reply:
`max()` works both when then Class definition has an explicit height and not an explicit height. It also works when resizing the browser window. Can you provide details on how you managed to not get it to work?

Reply:
When a Class definition has no explicit height given and the editor is empty, we get an scrollbar on the right side.  Tested with `Auto Messe Salzburg` (Event) on pimcore/demo. 

<img width="781" alt="image" src="https://github.com/pimcore/pimcore/assets/11687066/2265d782-fba3-4826-9fc7-c4bf2c11582c">

Reply:
Can't reproduce your issue on demo.pimcore.fun.